Applications/Express.md

```diff
@@ -0,0 +1,76 @@
+## Express
+
+The express project is applying to become an incubating
+[Top Level Project of the Node.js Foundation][].
+
+### About Express, User base, Community and Ecosystem
+
+Express is one of the most popular web server frameworks available for Node.js.
+For many Node.js developers, express is their primary interface with Node.js
+on the server.
+
+Express has consistently been among the top packages installed via npm, with
+nearly 4.5 million downloads in just the past month.
+
+It is important to recognize that Express is not the only web framework in the
+Node.js ecosystem and that by incubating Express, neither the Node.js
+Foundation or Node.js core would be "picking a winner" or endorsing Express
+over any other overlapping or similar project.
+
+### Formation of an Express WG
+
+The express project consists of a single top level core repository
+(strongloop/express) that has dependencies on a large number of smaller
+independent components that are currently spread out across multiple
+independent GitHub repositories. This proposal moves only the
```

Comment:
@mikeal The official express middleware (i.e. what used to be bundled in express <4) + maybe a few other things